src/ThemeProvider/ThemeProvider.js

```diff
@@ -51,11 +51,22 @@ const theme = createTheme({
   }
 });
 
+// custom theme for dark mode
+const DarkTheme = createTheme({
+  palette: {
+    mode: "dark"
+  }
+});
+
 /**
  * IPG Material-ui theme provider
  */
-export default function ThemeProvider({ children }) {
-  return <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>;
+export default function ThemeProvider({ children, mode }) {
```

Comment:
set default to light so that if ThemeProvider is called without this new mode prop then we still get same behaviour as today with default being light.

```suggestion
export default function ThemeProvider({ children, mode = "light" }) {
```